cfgov/v1/jinja2/v1/includes/organisms/contact-preview.html

```diff
@@ -0,0 +1,9 @@
+{% extends "v1/layouts/layout-full.html" %}
```

Comment:
@anselmbradford I was wondering whether there might be an easy way to preview an individual module without bringing in the whole base template including header, footer, and mega menu. For my template debug views (e.g. http://localhost:8000/admin/template_debug/v1/table/) I tried explicitly excluding the [header](https://github.com/cfpb/consumerfinance.gov/blob/05731c677d1a487e5bae06f1cf323a304d6c913a/cfgov/v1/jinja2/v1/template_debug.html#L29) and [footer](https://github.com/cfpb/consumerfinance.gov/blob/05731c677d1a487e5bae06f1cf323a304d6c913a/cfgov/v1/jinja2/v1/template_debug.html#L60) blocks but this seems suboptimal (and also causes a browser error because the base template tries to initialize the header which isn't rendered).

If I try to use a standalone template instead then things don't render correctly because we don't load our global page styles (and I'm not sure if this content needs to be in some sort of container as well).

Could we consider creating an even baser base template -- one that includes styles and some global structure but doesn't include the cf.gov header/footer chrome -- and then have the actual page base inherit from that? It might not be worth it for this one use case but wondering if you can think of any other cases it might be useful.

Reply:
I have an old issue somewhere about allowing the header/footer to be optional. I think the original use-case was the complaint intake portal. I think a more basic template makes sense for your needs.